include/gz/sim/rendering/SceneManager.hh

```diff
@@ -118,6 +118,12 @@ inline namespace GZ_SIM_VERSION_NAMESPACE {
     /// \return World ID
     public: Entity WorldId() const;
 
+    /// \brief Set the spherical coordinates from world.
+    /// \param[in] _sphericalCoordinates SphericalCoordinates
+    /// from the world.
+    public: void SetSphericalCoordinates(
+        math::SphericalCoordinates _sphericalCoordinates);
```

Comment:
We should always pass non-POD arguments as const-ref to prevent unnecessary copies

```suggestion
        const math::SphericalCoordinates &_sphericalCoordinates);
```

Reply:
Done https://github.com/gazebosim/gz-sim/pull/1556/commits/c5ea0cb0204a46199f2c5be54da3361fb65f7b81